packages/common/src/utils/dateUtils.ts

```diff
@@ -124,11 +124,11 @@ const getIsIntervalsCorrect = (leftStartTime: Date, leftEndTime: Date, rightStar
 
 export const getPeriodWorkHours = (hoursWorkingPerWeek: number, sickLeavePercentage: string) => {
   if (sickLeavePercentage === SickLeavePeriods.EN_FJARDEDEL) {
-    return (hoursWorkingPerWeek *= 0.75)
+    return (hoursWorkingPerWeek *= 0.75).toLocaleString('sv-SE')
```

Comment:
This can be extracted to a function so you don't have do add functionality in three places. You can send in hoursWorkingPerWeek and the percentage constant as parameters